masterlist.yaml

```diff
@@ -21732,3 +21732,20 @@ plugins:
     msg: [ *alreadyInOrFixedByBDSv3_0 ]
 
 ###### Anything new can go after this (If you're not sure where to put it) ######
+
+  # Expanded Tel Mithryn
+  - name: 'telmith.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/18956' ]
+    dirty:
+      # version 1.2
```

Comment:
Comments like these aren't necessary as the version number will be included in the commit message:

```suggestion
```